client/nginx_test.go

```diff
@@ -224,9 +235,227 @@ func TestStreamDetermineUpdates(t *testing.T) {
 	}}
 
 	for _, test := range tests {
-		toAdd, toDelete := determineStreamUpdates(test.updated, test.nginx)
+		toAdd, toDelete, _ := determineStreamUpdates(test.updated, test.nginx)
 		if !reflect.DeepEqual(toAdd, test.expectedToAdd) || !reflect.DeepEqual(toDelete, test.expectedToDelete) {
 			t.Errorf("determiteUpdates(%v, %v) = (%v, %v)", test.updated, test.nginx, toAdd, toDelete)
 		}
 	}
 }
+
+func TestDetermineUpdatesWithUpdate(t *testing.T) {
+	maxConns := 1
+	var tests = []struct {
+		updated          []UpstreamServer
+		nginx            []UpstreamServer
+		expectedToUpdate []UpstreamServer
+	}{{
```

Comment:
All beginning and end of `DetermineUpdates` test structs are strange. Can you take a look?

```suggestion
	}{
	    {
```

Reply:
It is done like that in the other methods https://github.com/nginxinc/nginx-plus-go-client/blob/master/client/nginx_test.go#L14

I just didn't changed it because everything was done in that way for consistency, I can fix everything if needed.